.github/workflows/pulumify.yml

```diff
@@ -11,6 +11,6 @@ jobs:
         AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
         GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         PULUMI_ACCESS_TOKEN: ${{ secrets.PULUMI_ACCESS_TOKEN }}
-        PULUMIFY_BUILD: make ensure && hugo --minify --buildFuture -e $GITHUB_SHA
+        PULUMIFY_BUILD: NODE_ENV=production make ensure && NODE_ENV=production hugo --minify --buildFuture -e $GITHUB_SHA
```

Comment:
Not sure why this isn't just settable at the action's `env` level, but it doesn't seem to be getting applied. Could have something to do with the way the command is being executed by Pulumify, too. 🤔

Reply:
Yep, [it does](https://github.com/pulumi/actions-pulumify/blob/0781e889546d5cce5efcb3eec289894fb043e841/infra/pulumify#L82-L85).